src/cache/precomputed_quantities.jl

```diff
@@ -4,6 +4,14 @@
 import Thermodynamics as TD
 import ClimaCore: Spaces, Fields
 
+e_tot(ᶜY) = ᶜY.ρe_tot / ᶜY.ρ
+n_liq(ᶜY) = ᶜY.ρn_liq / ᶜY.ρ
```

Comment:
We should also have `ᶜY.ρn_rai` there somewhere

Reply:
It may be that we don't use the divided by rho version anywhere. But lets define the divide by function anyway, if we can.